Automattic/wp-calypso
# Implement the Logged out in-page CTA section

#### Proposed Changes
More info: pdgAPm-yE#comment-639-p2
Figma file: 2ea28-pb
This implements the in-page CTA section for Logged Out Plugins section. This is visible on `/plugins`.

<img width="320" src="https://user-images.githubusercontent.com/2749938/212348078-cf5358af-bdac-4a75-a8d2-f6790c0b7a40.jpeg" />
<img width="320" src="https://user-images.githubusercontent.com/2749938/212348072-4f9d917d-759d-4b44-ba56-3bfc84407a0c.jpeg" />

#### Testing Instructions



* Log out
* Go to /plugins
* The in-page CTA section should be visible
<img width="480" alt="Screenshot 2023-01-13 at 16 48 34" src="https://user-images.githubusercontent.com/2749938/212348468-3508000f-3492-46f2-9fa5-02bfde716ca8.png">
<img width="240" alt="Screenshot 2023-01-13 at 16 49 03" src="https://user-images.githubusercontent.com/2749938/212348454-3669f21b-403c-4947-8cdd-de1b553f61b1.png">

* Makes sure that the CTA button points to the Business plan flow and that a `calypso_plugin_in_page_cta_click` event is triggered on click,

* Log in
* Go to /plugins
* The in-page CTA section should not be visible



#### Pre-merge Checklist





Related to #
